### PR TITLE
Feature/ms 478/direct text highlight

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
# Pull request summary

Related to [MS-478](https://oat-sa.atlassian.net/browse/MS-478).

Direct text highlight

# How to test

Check all the task AC-s are met

### CODE REVIEW

File: views/js/previewer/plugins/content/scoringHighlighter.js [hunk 1]
```diff
@@ -170,6 +180,74 @@ define([
                 }
             };
 
+            /**
+             * Turns on the highlighter and adds the cursor
+             */
+            this.turnHighlighterOn = () => {
+                const $container = $(CONTAINER_SELECTOR);
+
+                PointerEvent = (window.PointerEvent || window.MSPointerEvent);
```
Comment (ampaveliev) [inline views/js/previewer/plugins/content/scoringHighlighter.js]: global variable?

Comment (ampaveliev) [reply views/js/previewer/plugins/content/scoringHighlighter.js]: does it necessary to use MSPointerEvent?

File: views/js/previewer/plugins/content/scoringHighlighter.js [hunk 2]
```diff
@@ -38,11 +38,15 @@ define([
          */
         init() {
             const testRunner = this.getTestRunner();
+
             this.selection = window.getSelection();
-            
+
             this.isEraserOn = false;
+            this.isHighlighterOn = false;
```
Comment (ampaveliev) [inline views/js/previewer/plugins/content/scoringHighlighter.js]: Should it be value instead of boolean? We are planning to use multiple colors in future

**Review by ampaveliev — COMMENTED** [review]
Works cool, but I have some comments

File: views/js/previewer/plugins/content/scoringHighlighter.js (hunk 2 above)
Comment (AnastasiaArcadia) [reply views/js/previewer/plugins/content/scoringHighlighter.js]: This var was unused, removed it

File: views/js/previewer/plugins/content/scoringHighlighter.js [hunk 3]
```diff
@@ -160,8 +169,8 @@ define([
 
             /**
              * Toggles the eraser mode
-             * 
-             * @param {Boolean} isOn 
+             *
+             * @param {Boolean} isOn
```
Comment (ampaveliev) [inline views/js/previewer/plugins/content/scoringHighlighter.js]: no params

**Review by dresha-dev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by ampaveliev — APPROVED** [review]
works like a charm. gj!

**Pull request merged**